docs/sdk/wallets/minting-methods/mint-via-contracts.md

```diff
@@ -0,0 +1,122 @@
+# Mint via Contracts
+
+You can mint a PKP NFT from the PKP contract on Chronicle - Lit's custom EVM rollup testnet - using 
+1. The [Lit explorer](https://explorer.litprotocol.com/mint-pkp), 
+2. The Lit relayer (sign up for an API key [here](https://forms.gle/RNZYtGYTY9BcD9MEA)) or 
+3. The contracts directly using the [contracts-sdk](https://js-sdk.litprotocol.com/modules/contracts_sdk_src.html). 
+
+The NFT represents root ownership of the PKP. The NFT owner can grant other users (via a wallet address) or grant Lit Actions the ability to use the PKP to sign and decrypt data. They also have the ability to assign additional authentication methods, described at the bottom of the page.
+
+You can also use the handy helper contract on Chronicle [here](https://chain.litprotocol.com/address/0xDe905Fde36562270AA6FEeBAbC5aB1f440f733c2) to mint and assign auth methods, as well as view all of the deployed contract addresses [here](https://github.com/LIT-Protocol/networks/tree/main/cayenne).
+
+## Installing the required packages
+```bash
+yarn add @lit-protocol/lit-auth-client
+yarn add @lit-protocol/contracts-sdk
+```
+
+## Initializing your `LitContract` instance
+```js
+import { LitContracts } from '@lit-protocol/contracts-sdk';
+
+// if no signer is provided, it will attempt to use window.etheruem
+const contractClient = new LitContracts({ signer });
+await contractClient.connect();
+```
+
+## Minting a PKP and adding permitted scopes
```

Comment:
can we add a few sentences of context that introduce permitted scopes and why they are required?

Reply:
Added @a1ttech :)